### PR TITLE
Shared Preferences fix 

### PR DESCRIPTION
**Pull request by MaksPetrukhnoPI**

### CODE REVIEW

**Review by ValentinPostindustria — APPROVED** [review]
LGTM!

File: PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/sdk/deviceData/managers/UserConsentManager.java [hunk 1]
```diff
@@ -113,7 +113,15 @@ private void updateConsentValue(
 
         switch (key) {
             case GDPR_PREBID_SUBJECT:
-                gdprPrebidSubject = preferences.getString(GDPR_PREBID_SUBJECT, null);
+                try {
+                    gdprPrebidSubject = preferences.getString(GDPR_PREBID_SUBJECT, null);
+                } catch (ClassCastException classCastException) {
```
Comment (YuriyVelichkoPI) [inline PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/sdk/deviceData/managers/UserConsentManager.java:118]: Need to polish the code here:
- Use curly brackets for if-else 
- Add a comment to the line where we try to extract the boolean value. Mention the bug in the comment
- wrap getBoolean into try-catch as well. Who knows what we can meet there. 
- keep the consistent formatting for `preferences.edit()` line - put each function on the separate row

**Review by YuriyVelichkoPI — CHANGES_REQUESTED** [review]

**Review by YuriyVelichkoPI — APPROVED** [review]
LGTM

**Review by ValentinPostindustria — APPROVED** [review]
LGTM

**Pull request merged**